synthetic_data/marginal_dist.py

```diff
@@ -67,7 +70,7 @@ def _detect_dist_discrete(col_stats):
     # can sort if all numeric here
     try:
         mapping_order = np.argsort(list(map(int, categories)))
-    except:
+    except:  # noqa E722
```

Comment:
I recommend updating this so that it is not a bare execption

Reply:
`except Exception:` would work, but let's put a `TODO` and an issue on the repo to investigate